### ISSUE TITLE
Как выделить месяцы календаря?

### ISSUE THREAD

**Issue by a-files** [t1m0n/air-datepicker#210]
Добрый день!
Подскажите как выделить месяцы при клике на поле, если они были записаны с помощью разделителя?
Вот код:
`
this.inputSeasonMonth.not('[disabled]').datepicker(
{
    autoClose : false,
    classes : 'no-select',
    clearButton : true,
    dateFormat : 'm',
    language : LANGUAGE,
    minView : 'months',
    position : 'bottom center',
    maxDate : new Date(DATE.getFullYear(), 11, 31, 0, 0, 0, 0),
    minDate : new Date(DATE.getFullYear(), 0, 1, 0, 0, 0, 0),
    multipleDates : 11,
    multipleDatesSeparator : ';',
    view : 'months',
    onSelect : function(formattedDate, date, inst) {
     selectMonth(inst.$el); // Добавляются концевые разделители ";"
    }
   }).data('datepicker');
`

Месяцы записываются в формате, например: ;1;2;3;7;8;9;
Пользователю не удобно заново выбирать их, нужно запоминать что было
Спасибо

**Comment by t1m0n** [comment]
Добрый день!
Не совсем понял вопроса. Есть строка формата `var months = '1;2;3;7;8;9'`, нужно получить из нее данные и отобразить на календаре?

**Comment by a-files** [comment]
Да, отобразился календарь, пользователь выбрал месяцы. Сохранил. Перечитал страницу. Нажимает на поле, а выбранные месяцы не подсвечены.
Как сделать, чтобы они были выделены на календаре
